src/Contracts/KubeServiceBase.ts

```diff
@@ -16,6 +16,13 @@ export enum KubeResourceType {
     StatefulSets = 32,
 }
 
+export enum KubeServiceType {

```

Comment:
do we need to hard code these? can not we take from the list of items we have? is it not that spec says so?

KubeServiceType

also please use double quotes rather single quote for consistency purpose across the code.

"javascript.preferences.quoteStyle": "double"

you can use above options in your vscode settings for this purpose

Reply:
Addressed it. Removed `KubeServiceType` and picked the types to display from `serviceList`